source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -10,18 +13,20 @@ CLSBeamlineStatus::CLSBeamlineStatus(const QString &name, QObject *parent) :
 	monoMaskControlSet_ = new AMControlSet(this);
 
 	// Setup the basic value options.
-	addOption(On, "On", true);
-	addOption(Off, "Off", true);
+	addOption(On, "Beam is ON", true);
+	addOption(Off, "Beam is OFF", true);
+
+	connect(this, SIGNAL(connected(bool)), this, SLOT(updateBeamlineStatus()));
```

Comment:
I'm always a little leery about connecting our own signals to our own slots. Is it possible to make the calls to `updateBeamlineStatus()` in the places where these signals are originally emitted?